Application/EdFi.Ods.Api/Security/Authorization/EntityAuthorization/EntityAuthorizer.cs

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using EdFi.Common.Utils.Extensions;
+using EdFi.Ods.Api.Security.Authorization.Filtering;
+using EdFi.Ods.Api.Security.Authorization.Repositories;
+using EdFi.Ods.Common.Infrastructure.Filtering;
+using EdFi.Ods.Common.Security.Authorization;
+using EdFi.Ods.Common.Security.Claims;
+using log4net;
+
+namespace EdFi.Ods.Api.Security.Authorization.EntityAuthorization;
+
+public class EntityAuthorizer : IEntityAuthorizer
+{
+    private readonly IDataManagementAuthorizationPlanFactory _dataManagementAuthorizationPlanFactory;
+    private readonly IEntityInstanceDelegateFilterAuthorizer _entityInstanceDelegateFilterAuthorizer;
+    private readonly IEntityInstanceAuthorizationValidator _entityInstanceAuthorizationValidator;
+    private readonly IEntityInstanceViewBasedFilterAuthorizer _entityInstanceViewBasedFilterAuthorizer;
+
+    private readonly ILog _logger = LogManager.GetLogger(typeof(EntityAuthorizer));
```

Comment:
This logger is unused and could be removed.

Reply:
Removed.